### ISSUE TITLE
Skip all tags, when skip_tags = ""

### ISSUE THREAD

**Issue by hituzi-no-sippo** [orhun/git-cliff#136]
**Describe the bug**

Skip all tags, when `skip_tags = ""`.

**To Reproduce**

I've created a test script.

<details><summary>test script</summary>

```bash
#!/bin/bash

set -e

rm -rf /tmp/cliff/skip_tags_test
mkdir -p /tmp/cliff/skip_tags_test
cd /tmp/cliff/skip_tags_test

git init

git commit --allow-empty -m "chore: v0.0.1"
git tag v0.0.1 --message v0.0.1

git commit --allow-empty -m "feat: v0.1.0-beta.1"
git tag v0.1.0-beta.1 --message v0.1.0-beta.1

git commit --allow-empty -m "feat: v0.1.0"
git tag v0.1.0 --message v0.1.0

git cliff --version

git cliff --init

echo '====================================='
echo 'skip_tags = "v0.1.0-beta.1" (Default)'
echo '====================================='
git cliff

sed -i 's/\(skip_tags = \)"v0.1.0-beta.1"/\1""/' cliff.toml
echo '====================================='
echo 'skip_tags = ""'
echo '====================================='
git cliff

sed -i 's/skip_tags.*//' cliff.toml
echo '====================================='
echo 'skip_tags is unset.'
echo '====================================='
git cliff
```

</details>

<details><summary>output</summary>

```
Initialized empty Git repository in /tmp/cliff/skip_tags_test/.git/
[main (root-commit) 48e1819] chore: v0.0.1
[main 7b79e50] feat: v0.1.0-beta.1
[main e1ca355] feat: v0.1.0
git-cliff 1.1.1
 INFO  git_cliff > Saving the configuration file to "cliff.toml"
=====================================
skip_tags = "v0.1.0-beta.1" (Default)
=====================================
# Changelog

All notable changes to this project will be documented in this file.

## [0.1.0] - 2023-01-13

### Features

- V0.1.0

## [0.0.1] - 2023-01-13

### Miscellaneous Tasks

- V0.0.1

<!-- generated by git-cliff -->
=====================================
skip_tags = ""
=====================================
# Changelog

All notable changes to this project will be documented in this file.

<!-- generated by git-cliff -->
=====================================
skip_tags is unset.
=====================================
# Changelog

All notable changes to this project will be documented in this file.

## [0.1.0] - 2023-01-13

### Features

- V0.1.0

## [0.1.0-beta.1] - 2023-01-13

### Features

- V0.1.0-beta.1

## [0.0.1] - 2023-01-13

### Miscellaneous Tasks

- V0.0.1

<!-- generated by git-cliff -->
```

</details>

`skip_tags = ""`

```
# Changelog

All notable changes to this project will be documented in this file.

<!-- generated by git-cliff -->
```

**Expected behavior**

I expect don't skip all tags, as when `skip_tags` is unset.

`skip_tags` is unset.

```
# Changelog

All notable changes to this project will be documented in this file.

## [0.1.0] - 2023-01-13

### Features

- V0.1.0

## [0.1.0-beta.1] - 2023-01-13

### Features

- V0.1.0-beta.1

## [0.0.1] - 2023-01-13

### Miscellaneous Tasks

- V0.0.1

<!-- generated by git-cliff -->
```

**System (please complete the following information):**

- OS Ubuntu 20.04 LTS (WSL2)
- Project Version: [v1.1.1](https://github.com/orhun/git-cliff/releases/tag/v1.1.1)

**Additional context**

It seems to be caused by 8b17a1f02619027bebc5df1a8938aaf76adcd631.
The bug occurs in 8b17a1f02619027bebc5df1a8938aaf76adcd631, but not in ce1b7c3d7b7622af03994bd7a2fbcbb00a7be8bf.

Thank you.

**Comment by orhun** [comment]
Hey, thank you for reporting! 🐻

Indeed, this is is a regression in 8b17a1f02619027bebc5df1a8938aaf76adcd631 and should be fixed via eee35ad1eeeb89313e69086ad265ab400a6b4898.

**Comment by hituzi-no-sippo** [comment]
Good job 👍

I've checked, it worked as expected.
Thank you 😄
